network-mux/demo/mux-demo.hs

```diff
@@ -28,7 +32,7 @@ import Data.Bits
 import System.IOManager
 import System.Win32
 import System.Win32.Async qualified as Win32.Async
-import System.Win32.NamedPipes
+import "Win32-network" System.Win32.NamedPipes
```

Comment:
Why do we need package imports here?

Reply:
Because some of your changes got merged onto most recent "Win32" and there's conflicts on the import names

Reply:
So proper way forward is to update `win32-network` to a newer `Win32` version.